client/src/components/Login/index.jsx

```diff
@@ -17,24 +40,26 @@ export default class extends Component {
             <span className="login__logo-text login__logo-text--white">UN</span>
             <span className="login__logo-text login__logo-text--black">OPS</span>
           </section>
-          <form action="" className="login__form">
+          <form action="" className="login__form" onSubmit={this.handelSubmit}>
             <label htmlFor="email" className="login__label">
               <span className="login__label-text">
                 Email:
                 {' '}
                 <span className="required">*</span>
               </span>
-              <input type="email" id="email" className="login__input" />
+              <input name="email" type="email" id="email" className="login__input" onChange={this.handelChange} />
             </label>
             <label htmlFor="password" className="login__label">
               <span className="login__label-text">
                 Password:
                 {' '}
                 <span className="required">*</span>
               </span>
-              <input type="password" id="password" className="login__input" />
+              <input type="password" id="password" className="login__input" name="password" onChange={this.handelChange} />
             </label>
-            <input type="button" className="login__submit" value="Login" />
+            <span className="login_msg">{this.state.msg}</span>
```

Comment:
the classname should be in double underscores.
```suggestion
            <span className="login__msg">{this.state.msg}</span>
```